src/entry.js

```diff
@@ -6,13 +6,11 @@ import { init } from './store';
 import App from './App';
 import logger from 'redux-logger';
 
-// exposes webpack variable RELEASE
-/*global RELEASE:true*/
-/*eslint no-undef: "error"*/
+const pathName = window.location.pathname.split('/');
 
 ReactDOM.render(
     <Provider store={init(logger).getStore()}>
-        <Router basename={ `/${RELEASE}/platform/inventory` }>
+        <Router basename={`${pathName[1] === 'beta' ? pathName[2] : pathName[1]}/inventory`}>
```

Comment:
@jharting this is how you can load inventory on multiple endpoints. It will use basename as either `rhcs/inventory` or `insights/inventory`.